docs/content/fsdocs-default.css

```diff
@@ -586,3 +592,41 @@ div.fsdocs-tip {
         color: #d1d1d1;
         font: 0.85rem 'Roboto Mono', monospace;
     }
+
+.button {
```

Comment:
Do you remember what these changes are for?  In general the fsdocs CSS doesn't give styline apart from specific fsdocs class and known element names

Reply:
This is for the install button we have on the home page. Can be removed if it doesn't apply to typical fsdocs usage.
![image](https://user-images.githubusercontent.com/5365982/126656369-90f9ffd0-fe11-4b66-a2cf-c43c69b56a54.png)

Reply:
Thanks, yes, removed that bit.